backend/src/main/java/com/bakdata/conquery/commands/DistributedStandaloneCommand.java

```diff
@@ -73,60 +70,26 @@ public void startStandalone(Environment environment, Namespace namespace, Conque
 		conquery.setManagerNode(managerNode);
 		conquery.run(manager);
 
-		//create thread pool to start multiple ShardNodes at the same time
-		ExecutorService starterPool = Executors.newFixedThreadPool(
-				config.getStandalone().getNumberOfShardNodes(),
-				new ThreadFactoryBuilder()
-						.setNameFormat("ShardNode Storage Loader %d")
-						.setUncaughtExceptionHandler((t, e) -> {
-							ConqueryMDC.setLocation(t.getName());
-							log.error("{} failed to init storage of ShardNode", t.getName(), e);
-						})
-						.build()
-		);
-
-		List<Future<ShardNode>> tasks = new ArrayList<>();
-		for (int i = 0; i < config.getStandalone().getNumberOfShardNodes(); i++) {
-
-			final int id = i;
+		for (int id = 0; id < config.getStandalone().getNumberOfShardNodes(); id++) {
 
-			tasks.add(starterPool.submit(() -> {
-				ShardNode sc = new ShardNode(ShardNode.DEFAULT_NAME + id);
```

Comment:
@awildturtok 
Da die ClusterConnectionShard hier durch den lifecycle gemanaged wird, gab es hier eine Racecondition beim Hinzufügen der Connection zum lifecycle. Dadurch wurden beide Connections mal gemanaged, oft hats aber nur eine Connection geschafft hinzugefügt zu werden.

Im Standalone, werden die shards jetzt nacheinander instanziiert. Dadurch werden innerhalb eines Shards zwar noch die WorkerStorages parallel geladen, aber nicht mehr die WorkerStorages aller Shards parallel (ist verkraftbar denke ich).

Generell könnte das schon lange eine Quelle für andere spurious Fehler in der CI gewesen sein, da sich im StandaloneCommand ein environment/lifecycle für alle Komponenten geteilt wurde.